{
  "affected": [
    {
      "database_specific": {
        "source": "https://storage.googleapis.com/cve-osv-conversion/osv-output/CVE-2023-0509.json"
      },
      "ranges": [
        {
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "a9098bdf7406e6faf9df3da6ff2d584e90c13bbb"
            }
          ],
          "repo": "https://github.com/pyload/pyload",
          "type": "GIT"
        }
      ],
      "versions": [
        "v0.1",
        "v0.1.1",
        "v0.2",
        "v0.2.1",
        "v0.2.2",
        "v0.3",
        "v0.3.1",
        "v0.3.2",
        "v0.4",
        "v0.4.1",
        "v0.4.2",
        "v0.4.3",
        "v0.4.4",
        "v0.4.5",
        "v0.4.6",
        "v0.4.7",
        "v0.4.8",
        "v0.4.9"
      ]
    }
  ],
  "aliases": [
    "GHSA-8v53-23mx-hcf9"
  ],
  "details": "Improper Certificate Validation in GitHub repository pyload/pyload prior to 0.5.0b3.dev44.",
  "id": "CVE-2023-0509",
  "modified": "2025-10-10T04:21:52.567118Z",
  "published": "2023-01-26T22:15:26Z",
  "references": [
    {
      "type": "FIX",
      "url": "https://github.com/pyload/pyload/commit/a9098bdf7406e6faf9df3da6ff2d584e90c13bbb"
    },
    {
      "type": "FIX",
      "url": "https://huntr.dev/bounties/a370e0c2-a41c-4871-ad91-bc6f31a8e839"
    }
  ],
  "schema_version": "1.7.3",
  "severity": [
    {
      "score": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:H/I:H/A:N",
      "type": "CVSS_V3"
    }
  ]
}